exchanges/kraken/kraken_websocket.go

```diff
@@ -1248,7 +1248,7 @@ channels:
 		}
 
 		for j := range *s {
-			(*s)[j].Pairs = append((*s)[j].Pairs, channelsToSubscribe[i].Pair.String())
+			(*s)[j].Pairs = append((*s)[j].Pairs, channelsToSubscribe[i].Pairs[0].String())
```

Comment:
You've added configurable subscriptions to Kraken. Authenticated subscriptions do not require `Pairs`, this looks like a spot for trouble if I'm reading this correctly. Same for Unsub

Reply:
Yeah. Because I'd manhandled Kraken so much before abstracting this (#1516 - The revival of #1358) I was trying to be gentle here. Especially since this specific paradigm makes my brain hurt.
But nevermind the excuses.

Fixed c3250d3